### DISCUSSION TITLE
Equipos para el hackaton

### DISCUSSION THREAD

**Discussion by PacoMedGar** [Launch-X-Latam/MisionFrontEnd#367] [Q&A]
Alguien quiere empezar a hacer equipos para el hackaton?

**Comment by CarlosBinary** [comment]
¡Hola Paco!

Estaría encantado de integrarme a un equipo, si gustas y lo crees conveniente puedes contactarme.

**Comment by LaMazorca** [comment]
A mí me gustaría entrar, no tengo mucho que ofrecer, más que solo soy bueno para aprender

**Reply by CarlosBinary** [reply]
Yo encantado de formar grupo con ustedes!

**Comment by RafaelMendoza85** [comment]
Yo encantado de unirme a tu grupo si gustas

**Comment by SaulCADE25** [comment]
siguen buscando gente para su equipo o ya están completos
